src/GithubPlugin/Widgets/GithubIssuesWidget.cs

```diff
@@ -132,7 +132,7 @@ public override void LoadContentData()
                     { "title", issueItem.Title },
                     { "url", issueItem.HtmlUrl },
                     { "number", issueItem.Number },
-                    { "date", TimeSpanHelper.TimeSpanToDisplayString(DateTime.Now - issueItem.CreatedAt, Log.Logger()) },
+                    { "date", TimeSpanHelper.DateTimeOffsetToDisplayString(issueItem.CreatedAt, Log.Logger()) },

```

Comment:
I presume issueItem.CreatedAt is in UTC so that the new math works?

Reply:
Yes, I tested this here locally and it was showing the right time

Reply:
@dkbennett said all timestamps in the database are UTC.  So we should be fine here.